src/app/resources/curriculum/curriculum.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'variables';
+@import "variables";

```

Comment:
I know it's a bit confusing, but different file types have different standards. In this project the CSS files use single quotes '' instead of the double quotes.